thrift_medusa/vcs/GitVCS.py

```diff
@@ -129,10 +126,15 @@ def get_modified_files(self, current_commit=None, previous_commit=None, filter_d
             returns a list of all hashes (ie commits) between the two tags passed into the method.
 
             If current_commit or previous_commit is empty, will fall back on default behavior and return
-            list of modified files for latest commit.
+            list of modified files for and its parent commit.
```

Comment:
wording sounds weird.  for latest and its parent?  or for its parent commit?  the "and its parent" doesn't make sense.